source/connect/favorite-connections/import-export-cli/export.txt

```diff
@@ -0,0 +1,124 @@
+.. _compass-export-connections-cli:
+
+========================================
+Export Favorite Connections with the CLI
+========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can use the |compass-short| :abbr:`CLI (Command-Line Interface)` to
+export favorite connections. This lets you share saved connections
+between workspaces and with colleagues to connect to shared deployments.
+
+About This Task
+---------------
+
+When you export favorite connections, |compass-short| exports the list
+of connections as a JSON file.
+
+By default, |compass-short| does not hide sensitive connection
+information (such as usernames and passwords) from the exported
+connection file. To learn how to protect sensitive information in
+exported connections, see :ref:`compass-encrypt-connections-cli`.
+
+Procedure
+---------
+
+To export **unencrypted** favorite connections using the |compass-short|
```

Comment:
Consider:

... connections with the ...

Reply:
Done.